common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -75,6 +75,8 @@
   "feature.wynntils.chatMention.mentionColor.description": "What color should be used for highlighting?",
   "feature.wynntils.chatMention.mentionColor.name": "Highlight color",
   "feature.wynntils.chatMention.name": "Chat Mentions",
+  "feature.wynntils.chatMention.suppressMentionsInInfo.description": "Should there be mentions in info messages?",
```

Comment:
This is strangely worded. Remember that a true value means that they are suppressed. Maybe "Should mentions in informational system messages be ignored?"